packages/CollabCodeEdit-Tests.package/CCEClientMockTest.class/instance/testHasEditOfType.st

```diff
@@ -0,0 +1,9 @@
+tests
+testHasEditOfType
+	clientMock sendMessage: 
+		(CCEMessage sendEdit: 
+			(CCEMethodAddedOrModifiedEdit newWith: #testMethod in: CCETestDummy)).
+			
+	self assert: (clientMock hasEditOfType: CCEMethodAddedOrModifiedEdit).
+	self deny: (clientMock hasEditOfType: CCEMethodRecategorizedEdit) 
+	
```

Comment:
Why is there no `self deny: (clientMock hasEditOfType: CCEMethodRemovedEdit)`?

Reply:
In the future there will be even more CCEEdit classes. This serves merely as a sample type, which should not be in the clientMock.

Reply:
Makes totally sens now, leaf it as it is